packages/next/src/build/analysis/get-page-static-info.ts

```diff
@@ -444,29 +447,30 @@ function warnAboutExperimentalEdge(apiRoute: string | null) {
   apiRouteWarnings.set(apiRoute, 1)
 }
 
-const warnedUnsupportedValueMap = new LRUCache<string, boolean>({ max: 250 })
+export let hadUnsupportedValue = false
 
 function warnAboutUnsupportedValue(
   pageFilePath: string,
   page: string | undefined,
   error: UnsupportedValueError
 ) {
-  if (warnedUnsupportedValueMap.has(pageFilePath)) {
-    return
-  }
+  hadUnsupportedValue = true
 
-  Log.warn(
+  const message =
     `Next.js can't recognize the exported \`config\` field in ` +
-      (page ? `route "${page}"` : `"${pageFilePath}"`) +
-      ':\n' +
-      error.message +
-      (error.path ? ` at "${error.path}"` : '') +
-      '.\n' +
-      'The default config will be used instead.\n' +
-      'Read More - https://nextjs.org/docs/messages/invalid-page-config'
-  )
-
-  warnedUnsupportedValueMap.set(pageFilePath, true)
+    (page ? `route "${page}"` : `"${pageFilePath}"`) +
+    ':\n' +
+    error.message +
+    (error.path ? ` at "${error.path}"` : '') +
+    '.\n' +
+    'Read More - https://nextjs.org/docs/messages/invalid-page-config'
+
+  // for a build wait to log all errors before exiting
```

Comment:
Do we have a test for multiple config errors?

Reply:
Yea that's in this test suite `test/production/exported-runtimes-value-validation/index.test.ts`